src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -104,3 +128,9 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
 
 </body>
 </html>
+  
+\______
+//------
```

Comment:
As a good practice, every time you send out a Code Review, before you send the code review out, you SHOULD do a review of your code yourself first, to at least cleanup obvious things , style / format related things yourself.

Ideally, reviewer should only be providing feedback on the design / structure of the code, or ask question about the background etc, not small things you can fix yourself if you had taken the time to review your code once before sending it out.